src/lib.rs

```diff
@@ -588,9 +584,59 @@ impl HelperState {
     }
 }
 
+/// Finds the position of `--jobserver-auth=` flag in a given environment variable.
```

Comment:
I guess the change is intended to be minimal, but I wonder if the function could simply return the final slice needed (i.e. the value of the flag found) rather than a tuple`(arg, pos)` to then compute a length outside and then the slice and so on.

Reply:
Nice idea! Updated. Let me know if you have any concern :)

Reply:
Looks good, thanks! It is much simpler.